cmake/modules/FindPipewire.cmake

```diff
@@ -0,0 +1,53 @@
+#.rst:
+# FindPipewire
+# --------------
+# Finds the Pipewire library
+#
+# This will define the following variables::
+#
+#  PIPEWIRE_FOUND - system has the Pipewire library
+#  PIPEWIRE_INCLUDE_DIRS - the Pipewire include directory
+#  PIPEWIRE_LIBRARIES - the libraries needed to use Pipewire
+#  PIPEWIRE_DEFINITIONS - the definitions needed to use Pipewire
+#
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_PIPEWIRE libpipewire-0.3>=0.3.24 QUIET)
```

Comment:
you should probably check for libspa-0.2 here as well

Reply:
Added, thanks!